test/powershell/engine/ExecutionPolicy.GroupPolicy.Tests.ps1

```diff
@@ -0,0 +1,48 @@
+﻿##
+## PowerShell ExecutionPolicy GroupPolicy tests
+## These are Windows only tests
+##
+
+Describe "User group policy execution policy should work" -Tags 'Feature' {
```

Comment:
We are going to need tests that change machine state and these tests must always clean up afterwards, or will affect subsequent tests.  If AfterAll block is not reliable then maybe try/finally should be used.